docs/api-reference/geo-layers/tile-3d-layer.md

```diff
@@ -196,6 +196,13 @@ When [`picking`](/docs/developer-guide/custom-layers/picking.md) is enabled, `in
   - `url`: the url of the failed tile.
   - `message`: the error message.
 
+##### `getTileColor` (Function, optional)
+`getTileColor` is a function which allows to change color of mesh based on `tileHeader` object.
+The rgb color at the target, in `r, g, b`. Each component is in the 0-255 range.
```

Comment:
This is a function. State the argument that it receives, and the expected return type. Also note that it only works for I3S.

Reply:
Added